amundsen_application/static/js/ducks/tableMetadata/api/helpers.ts

```diff
@@ -21,7 +21,12 @@ export function getTableDataFromResponseData(responseData) {
  * Parses the response for table metadata to return the array of table owners
  */
 export function getTableOwnersFromResponseData(responseData) {
-  return responseData.owners;
+  // TODO: owner needs proper id, until then we have to remember that we are using display_name
+  const ownerObj = responseData.owners.reduce((obj, owner) => {
```

Comment:
I would rename these as `accObj` and `currentOwner`. It's easy to forget what order these come.

Also I wonder if changing from an array to object will affect render order. This may or may not be an issue.